pkg/cliplugins/workspace/cmd/cmd.go

```diff
@@ -35,8 +35,8 @@ var (
 	# enter a given workspace (this will change the current-context of your current KUBECONFIG)
 	%[1]s workspace use my-workspace
 
-	# list all your personal workspaces
-	%[1]s workspace list
+	# list sub-workspaces in the current workspace 
+	%[1]s get workspaces
```

Comment:
@wangke19 you can submit a minor PR for these lines to fix your https://github.com/kcp-dev/kcp/issues/1621